load_db/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.6-stretch
 
 ENV MAIN_DIR=/srv/import_data \
     SQL_DIR=/srv/import_data/sql \
-    DATA_DIR=/data
+    DATA_DIR=/tmp/pbfs
```

Comment:
This would break the current docker-compose, unless you update the volume in "docker-compose.yml"